squarelet/users/fe_api/serializers.py

```diff
@@ -0,0 +1,96 @@
+# Third Party
+from allauth.socialaccount.models import SocialAccount, SocialToken
+from rest_framework import serializers
+
+# Squarelet
+from squarelet.organizations.serializers import MembershipSerializer
+from squarelet.users.models import User
+
+
+class SocialTokenSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SocialToken
+        fields = ("token",)
+
+
+class SocialAccountSerializer(serializers.ModelSerializer):
+    tokens = SocialTokenSerializer(many=True, source="socialtoken_set")
+    extra_data = serializers.JSONField()
+
+    class Meta:
+        model = SocialAccount
+        fields = ("provider", "uid", "extra_data", "tokens")
+
+
+class UserSerializer(serializers.ModelSerializer):
+    uuid = serializers.UUIDField(read_only=True)
+    username = serializers.CharField(read_only=True)
+    picture = serializers.CharField(source="avatar_url", read_only=True)
+    email_verified = serializers.SerializerMethodField()
+    email = serializers.SerializerMethodField()
+    organizations = serializers.SerializerMethodField()
+    social_accounts = SocialAccountSerializer(many=True, source="socialaccount_set")
+
+    class Meta:
+        model = User
+        fields = (
+            "id",
+            "uuid",
+            "username",
+            "name",
+            "email",
+            "email_failed",
+            "email_verified",
+            "is_agency",
+            "organizations",
+            "bio",
+            "picture",
+            "updated_at",
+            "use_autologin",
+            "social_accounts",
+        )
+        read_only_fields = fields
+
+    def get_primary_email_field(self, obj, field, default):
```

Comment:
A lot of this is going to be slow - I think starting simple is going to be beneficial here.  We can just use the `email` field directly for email, then we lose `email_verified` but I think that is ok.